doc/man1/enc.pod

```diff
@@ -199,12 +199,12 @@ encrypting a file and read from the encrypted file when it is decrypted.
 
 Some of the ciphers do not have large keys and others have security
 implications if not used correctly. A beginner is advised to just use
-a strong block cipher in CBC mode such as bf or des3.
+a strong block cipher in CTR mode such as AES or Camellia.
```

Comment:
I suggested CTR to avoid the padding attacks that can be effective against CBC.  CTR is supposed to be immune to these.  We're dealing with _beginners_ here so overstating things and erring on the side of safety is probably a good idea.  Thus CTR not CBC.  If GCM or CCM were available, I'd have suggested them instead.

I also thought about leaving just AES, in fact that was my original rewrite of this bit.  After a short discussion with @jon-oracle, I added Camellia to better match what was there (i.e. two suggestions rather than one).  This provides some coverage if either cipher is broken in the short to mid term future.  There is an attack against AES that reduces the search space by a factor of four -- clearly not practical yet but it will only improve.  We'll really be in trouble if AES falls quickly.